e2e/libs/page-events-helper.ts

```diff
@@ -76,7 +76,9 @@ export const shouldLogResponse = (request: Request): boolean => {
     '/data-set/',
     '/data-sets/',
     '/resources',
-    '/concept-sets/'
+    '/concept-sets/',
+    '/institutions',
+    '/admin/users/list'
```

Comment:
Exclude two additional response bodies:  Large bodies, made test log very hard to read.